packages/reactotron-app/App/Foundation/ConnectionSelector.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import { inject, observer } from 'mobx-react'
+
+@inject('session')
```

Comment:
This is a component of sorts that connects to `mobx`. It feels a little bit dirty but it buys us the ability to just dump this component on screen and pretend everything is fine (mainly because everything is fine...)

Reply:
That's fine.